.travis.yml

```diff
@@ -11,18 +12,39 @@ install: true
 cache:
   directories:
     - node_modules
+    
 jobs:
   include:
-    - stage: "Prepare Dependencies"
+    - stage: Mac Setup and Tests
+      os: osx
+      name: "Prepare Dependencies"
+      script:
+        - npm install
+        - npm update
+    - <<: *Mac-Setup-and-Tests
```

Comment:
Can I clarify the meaning of the `<<:` syntax ? Having some trouble finding it in the Travis CI reference docs

Reply:
`<<:` adds the content of the stage to another stage iirc